sdk/iot/provisioning/readme.md

```diff
@@ -0,0 +1,45 @@
+# Azure IoT Clients for Embedded C: Azure IoT Device Provisioning Client
+
+Official Embedded C client for Azure IoT Provisioning Service.
+
+For more details about Provisioning Service check [Azure documentation](https://docs.microsoft.com/azure/iot-dps/)
+
+## Getting started
+
+- Pelase refer to the state machine diagram which describes the Azure SDK for Embedded C flow for Provisioning Service [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service). 
+
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/provisioning/samples) explains samples for the Azure SDK for Embedded C IoT Provisioning Client and how to use them.
+
+## Need help?
+
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-c/issues/new/choose).
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure+c) or ask new ones on StackOverflow using
+  the `azure` and `c` tags.
+
+## Contributing
```

Comment:
The contributing section should be a reference to the repo contributing section.  That way we only need to update one location.

```
## Contributing

If you'd like to contribute to this library, please read the [contributing guide][azure_sdk_for_c_contributing] to learn more about how to build and test the code.

### License

Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.

<!-- LINKS -->
[azure_sdk_for_c_contributing]: https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md
[azure_sdk_for_c_license]: https://github.com/Azure/azure-sdk-for-c/blob/master/LICENSE
```

Reply:
done!